src/components/GridContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+import propTypes from 'prop-types';
+
+const GridContainerDiv = styled.div`

```

Comment:
Please remember to create single source of truth for things like font, sizes etc